synapse/lib/stormtypes.py

```diff
@@ -830,35 +827,32 @@ def addLibFuncs(self):
             'list': self._methList,
         })
 
-    def _reqAllowed(self, perm, name):
-        self.runt.allowed(perm, name)
-
     def _reqStr(self, name):
         if not isinstance(name, str):
             mesg = 'The name of a persistent variable must be a string.'
             raise s_exc.StormRuntimeError(mesg=mesg, name=name)
 
     async def _methGet(self, name, default=None):
         self._reqStr(name)
-        self._reqAllowed('storm:globals:get', name)
+        self.runt.reqAllowed(('storm:globals:get', name))
```

Comment:
I almost wonder if we want to make these hierarchical while we're in here - its a undocumented feature ATM